l10n-screenshots-config.yml

```diff
@@ -0,0 +1,75 @@
+locales:
```

Comment:
This file is needed later on to check that the number of screenshots taken is correct and also to know and select the available locales